qiskit_experiments/characterization/qubit_spectroscopy.py

```diff
@@ -377,6 +385,11 @@ def __init__(
 
         super().__init__([qubit])
 
+        self.set_analysis_options(
```

Comment:
I don't think this should be a part of analysis options, but I can understand the downside of adding calibration options to base experiment, or creating calibration experiment subclass. We need more discussion about this.

Reply:
Looks like we can do this differently. See here: 61c0920